MQuince/MQuince.Integration.Services.Implementation/SftpService.cs

```diff
@@ -40,5 +40,39 @@ public bool SendFile(string fileName)
             }
 
         }
-    }
+
+        public bool SaveFile(string fileName)
+        {
+            SftpConfig config = new SftpConfig
+            {
+                Host = "192.168.1.12",
+                Port = 22,
+                UserName = "tester",
+                Password = "password"
+            };
+            using (var client = new SftpClient(config.Host, config.Port, config.UserName, config.Password))
+            {
+                // connect and log in
+                client.Connect();
```

Comment:
If you first check if client is NOT connected, and return false, you have less lines of code.
eg.
if (!client.IsConnected) return false;
//then you don't need to check if IsConnected
using(var FileStream...){.. return true;}